src/server/scripts/ExilesReach/zone_exiles_reach.cpp

```diff
@@ -1733,10 +1733,10 @@ enum LostExpeditionFollowerData
     ACTOR_ID_ALLIANCE_SURVIVOR                  = 69830,
     ACTOR_ID_HORDE_SURVIVOR                     = 76283,
 
-    ACTOR_INDEX_SURVIVOR_ZERO                   = 0,
-    ACTOR_INDEX_SURVIVOR_ONE,
-    ACTOR_INDEX_SURVIVOR_TWO,
-    ACTOR_INDEX_SURVIVOR_THREE,
+    ACTOR_INDEX_ZERO                            = 0,
```

Comment:
tbh we can drop those ACTOR_INDEXes and just use the value directly where we call it